plugins/cloudtrail/pkg/cloudtrail/extract.go

```diff
@@ -214,6 +214,10 @@ func getfieldStr(jdata *fastjson.Value, field string) (bool, string) {
 	switch field {
 	case "ct.id":
 		val := jdata.GetStringBytes("eventID")
+		if val == nil {
+			val = jdata.GetStringBytes("request-id")
+		}
```

Comment:
This situation should never happen, `eventID` is a mandatory field. Moreover, it seems the path is `requestID`:
![image](https://github.com/falcosecurity/plugins/assets/4520100/70cd710f-12f5-4596-b47e-e9de65ba19b0)

Reply:
the thing is that the s3 notification event that we are receiving is this one: https://docs.aws.amazon.com/AmazonS3/latest/userguide/ev-events.html 
The part inside the "detail" field, I've tried to map some of the fields to the ct fields but I can remove them

Reply:
The plugin is from collecting the audit logs from cloudtrail, we should never receive a direct S3 event. Maybe I'm wrong or there's something I don't get.

Reply:
I am working on this feature to receive also s3 notification events https://sysdig.atlassian.net/browse/SSPROD-34513
We need to modify the plugin to extract fields also from that kind of events, that is the reason of the change, I spoke with @jasondellaluce and since the change is little he was ok not creating a new plugin or a private plugin for this, but let me know if you agree
The fields we are interested in are the s3 ones, so I can remove the ct ones if they can cause issues